commands/fetch_templates.go

```diff
@@ -137,3 +184,34 @@ func createPath(relativePath string, perms os.FileMode) error {
 	err := os.MkdirAll(dir, perms)
 	return err
 }
+
+// canWriteLanguage tells whether the language can be processed or not
+// if overwrite is activated, the directory template/language/ is removed before to keep it in sync
```

Comment:
@rgee0 @johnmccabe  I don't know if overwrite should delete anything. It should probably dump over the top. Not sure?

Reply:
Delete is a strong action. I do think it important that the user knows exactly what the current state of their templates is and that they aren't able to 'accidentally' overwrite.

Reply:
I have updated it to no longer delete before `--overwrite`.